.env.sample

```
@@ -122,6 +122,19 @@ OIDC_DISPLAY_NAME=OpenID Connect
 # Space separated auth scopes.
 OIDC_SCOPES=openid profile email
 
+# To configure the GitHub integration, you'll need to create a GitHub App at
+# => https://github.com/settings/apps
+#
+# When configuring the Client ID, add a redirect URL under "Permissions & events":
+# https://<URL>/api/github.callback
+GITHUB_CLIENT_ID=get_a_key_from_slack
+GITHUB_CLIENT_SECRET=get_the_secret_of_above_key
+GITHUB_APP_NAME=get_the_name_of_registered_github_app
+GITHUB_APP_ID=get_the_id_of_registered_github_app
+GITHUB_APP_PRIVATE_KEY=get_the_private_key_of_registered_github_app
```

```suggestion
GITHUB_CLIENT_ID=
GITHUB_CLIENT_SECRET=
GITHUB_APP_NAME=
GITHUB_APP_ID=
GITHUB_APP_PRIVATE_KEY=
```

Will document setup separately, these placeholders aren't terribly useful